apps/sales/views.py

```diff
@@ -1456,7 +1455,110 @@ def put(self, request, pk, *args, **kwargs):
         ]
 
         return build_response(1, "Records updated successfully", custom_data, status.HTTP_200_OK)
+    
+#================================================================================================================================
+class DocumentGeneratorView(APIView):
```

Comment:
Why is this class in sales view?
Cant we move this to masters? if you want you can make a call to method by passing the required params

Reply:
no I'll moved it to masters for better understandings.

Reply:
moved into masters